### ISSUE TITLE
MATLAB comments not working correctly

### ISSUE THREAD

**Issue by enewton** [aramk/crayon-syntax-highlighter#145]
I'm trying to use the MATLAB language, which uses % to indicate start of comments for that line, just like // in C code.

I can see the regex for comments looks vaguely correct, however it appears to match a % to a Tag type, which isn't defined in the MATLAB syntax files.

i.e. If I choose red for Tag and green for Comments, then the following you would expect to be green:
% this is all comments

However, the % is red, and 'this is all comments' is black.

Other than that the plugin looks great.

**Comment by lettergram** [comment]
There is still an issue with this

**Comment by aramk** [comment]
Yeah I haven't had enough free time to look at all the issues. Would love PRs if anyone's willing!

**Comment by lettergram** [comment]
I think you just need to change the regex to read (%?) to read percentage sign, the period . might be the issue.

**Comment by neusgaten** [comment]
This is still a issue. Inline Matlab comments are only recognized when there is at least one character prior to the percent, otherwise it is erroneously seen as code.

**Comment by cminto** [comment]
Same as issue posted here as https://github.com/aramk/crayon-syntax-highlighter/issues/340 (point 1).
